packages/flame/test/components/component_lifecycle_test.dart

```diff
@@ -4,25 +4,36 @@ import 'package:flame_test/flame_test.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_test/flutter_test.dart';
 
-class _MyComponent extends Component {
+class LifecycleComponent extends Component {
```

Comment:
Why public? Shouldn't this be self contained in this test file?

Reply:
This component is also used in the `component_test.dart`. Technically, the two test files should be combined (since we decided on a 1 test file per source file policy), but we haven't gotten to that yet. Once the files are combined, this component can be made private once again.